web-ui/src/components/menu/Menu.jsx

```diff
@@ -461,16 +470,6 @@ function Menu() {
             >
               Profile
             </MenuItem>
-            {isAdmin && (
-              <MenuItem
-                onClick={() => {
-                  closeAvatarMenu();
```

Comment:
I didn't need the call to `closeAvatarMenu` above since the button is no longer in that menu.